SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+We support only the latest release of the okapi binaries, which is dependent on upstream vulnerabilities being patched. Prior to `1.4.0`, there is an upstream vulnerability in the oberon library.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.5.0   | :white_check_mark: |
+| 1.4.0   | :white_check_mark: |
+| < 1.4.0 | :x:                |
+
+## Reporting a Vulnerability
+
+Contact any of the developers directly.
```

Comment:
Might be best to either open an issue or give an email. This doesn't give info how to contact them directly.

Reply:
Let's have them open an issue.